tensorflow_lite_support/ios/task/processor/sources/TFLClassificationResult+Helpers.m

```diff
@@ -24,21 +24,15 @@ + (TFLClassificationResult *)classificationResultWithCResult:
   NSMutableArray *classificationHeads = [[NSMutableArray alloc] init];
   for (int i = 0; i < cClassificationResult->size; i++) {
     TfLiteClassifications cClassifications = cClassificationResult->classifications[i];
-    NSMutableArray *classes = [[NSMutableArray alloc] init];
+    NSMutableArray *categories = [[NSMutableArray alloc] init];
     for (int j = 0; j < cClassifications.size; j++) {
       TfLiteCategory cCategory = cClassifications.categories[j];
-
-      TFLCategory *resultCategory = [TFLCategory categoryWithCCategory:&cCategory];
-      [classes addObject:resultCategory];
+      [categories addObject:[TFLCategory categoryWithCCategory:&cCategory]];
     }
-    TFLClassifications *classificationHead = [[TFLClassifications alloc] init];
-    classificationHead.categories = classes;
-    classificationHead.headIndex = i;
-    [classificationHeads addObject:classificationHead];
+    [classificationHeads addObject:[[TFLClassifications alloc] initWithHeadIndex:i
```

Comment:
nit: You should keep an intermediate variable TFLClassifications *classificationHead to improve readability.

Reply:
> @priankakariatyml What's the source of the burger image? Is it released under a license such as CC0 that allows us to add it into the repo?

I have added an alpha channel of 1s to the existing burger.jpg in the repo using a python script. So it its okay.